WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -654,6 +654,10 @@ extension WooAnalyticsEvent {
             ])
         }
 
+        static func orderEditButtonTappedWhileDisabledForCurrencyConflict() -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .orderEditButtonTappedWhileDisabledForCurrencyConflict, properties: [:])
```

Comment:
Thanks for adding a Tracks event to see how many merchants are encountering this to have a better idea about the priority 👍